<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigGUI</class>
 <widget class="QDialog" name="ConfigGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="network_mapping">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Sim</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="localHost">
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">127.0.0.1</string>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="cursorPosition">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="localPort">
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">40100</string>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>GCS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="remoteHost">
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">127.0.0.1</string>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
       <property name="cursorPosition">
        <number>9</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="remotePort">
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">40200</string>
       </property>
       <property name="maxLength">
        <number>32767</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IP address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Side</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="channels_mapping">
     <item>
      <layout class="QVBoxLayout" name="cc_layout">
       <item>
        <layout class="QHBoxLayout" name="cc_channel_0" stretch="1,4">
         <item>
          <widget class="QLabel" name="cc_ch0">
           <property name="text">
            <string>CH0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cc_ch0_map">
           <item>
            <property name="text">
             <string>not used</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CH1-Aileron</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="sim_layout">
       <item>
        <layout class="QHBoxLayout" name="sim_channel_0" stretch="1,4">
         <item>
          <widget class="QLabel" name="sim_ch0">
           <property name="text">
            <string>CH0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="sim_ch0_map">
           <item>
            <property name="text">
             <string>not used</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CH1-Aileron</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigGUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigGUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
